src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin calendar-styles {
```

Comment:
You don't need to create a mixin for these styles as they are specific for the calendar, and you can't reuse them